package.json

```diff
@@ -45,22 +45,23 @@
   "devDependencies": {
     "@apollo/client": "^3.2.5",
     "@babel/cli": "^7.12.1",
+    "@babel/plugin-proposal-optional-chaining": "^7.12.7",
```

Comment:
Let's start using this in all zimlets instead of lodash/get
:smiling_face_with_three_hearts: